CMakeLists.txt

```diff
@@ -2,13 +2,13 @@
 # This code is licensed under the MIT license (MIT)
 # (http://opensource.org/licenses/MIT)
 
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.2)
 
 project(Unicorn)
 
 option(UNICORN_BUILD_DOCUMENTATION "Build UnicornRender documentation" OFF)
 option(UNICORN_BUILD_DEMOS "Build UnicornRender demo projects" ON)
-option(UNICORN_BUILD_TESTS "Build UnicornRender tests" ON)
+option(UNICORN_BUILD_TESTS "Build UnicornRender tests" OFF)
```

Comment:
I just noticed that this makes it so no tests are executed in CI. I think we should keep `ON` by default.

Reply:
I think we shouldnt because tests takes time when Im compiling on my pc. Better write once in CI script cmake -DUNICORN_BUILD_TESTS=ON, then every time when Im recmaking my own downloaded project. I think we need to provide project as simple as possible. If we are enabling tests. We also can enable doxygen documentation.

Reply:
doxygen may not be installed but tests are independent